content/03.results.md

```diff
@@ -100,51 +109,44 @@ Genotype information is not available in the SVPOP catalog hence genotyping perf
+Generally, more reads mapped to the *cactus graph* with high identity (Figure {@fig:panel3}a) and high mapping quality (Figure {@fig:panel3}b) than to the *VCF graph*.
+
+Next, we compared the SV genotyping performance of both graphs.
 We mapped short reads from the 11 non-reference strains to both graphs and called variants for each strain using the vg toolkit's variant calling module (see [Methods](#toil-vg-call)).
-In the absence of a gold standard, we evaluated each callset based on the alignment of reads to a *sample graph* constructed from the callset (see [Methods](#calling-and-genotyping-of-svs)).
-If a given callset is correct, we expect that reads from the same sample will be mapped confidently and with high identity to the corresponding sample graph.
-Therefore, we compared the average mapping quality and percent identity of the short reads on each sample graph (see Figures {@fig:4}a and b).
-Similar to the results of our mapping analysis above, the *cactus graph* clearly outperformed the *VCF graph* for strains in the *S. paradoxus* clade and performed slightly better for strains in the *S. cerevisiae* clade.
-Again, our measurements did not show a large difference between strains included in the graph and those that were excluded. For a direct comparison, see Figure {@fig:panel6} which shows results of the same experiment on graphs generated from all 12 strains.
+There is no gold standard call set for these sample, so we used an indirect measure of SV calling accuracy.
+We evaluated each call set based on the alignment of reads to a *sample graph* constructed from the call set (see [Methods](#calling-and-genotyping-of-svs)).
+If a given call set is correct, we expect that reads from the same sample will be mapped with high identity and confidence to the corresponding sample graph.
+Therefore, we compared the average percent identity and mapping quality of the short reads on each sample graph (Figures {@fig:4}a and b).
+Similar to the mappability results, the *cactus graph* clearly outperformed the *VCF graph* for strains in the *S. paradoxus* clade and performed slightly better for strains in the *S. cerevisiae* clade.
+While the higher percent identity shows that the *cactus graph* represents the reads better (Figures {@fig:4}a), the higher mapping quality confirms that this did not come at the cost of added ambiguity or a more complex graph (Figures {@fig:4}b).
+Our results did not show a substantial difference between strains included in the graph and those that were excluded.
+This suggests that two strains from each clade as well as the reference strain are sufficient to capture most of the genetic variation among all the strains.
+For a direct comparison, see Figure {@fig:panel6} which shows results of the same experiment on graphs generated from all 12 strains.
+
+[//]: # (The figure mentioned in the previous sentence doesn't seem to be included here.)
```

Comment:
Is it a problem if we reference a supplementary figure from the main text?

Reply:
No, I think it's fine to point to a supplementary figure from the main text.

Reply:
Yeah, I guess that was my bad